package.json

```diff
@@ -51,8 +51,11 @@
     "karma-webpack": "^1.7.0",
     "legacy-loader": "0.0.2",
     "less": "^2.3.1",
+    "node-sass": "^3.4.2",
```

Comment:
I thought peer dependencies might be involved :smile_cat:

I basically just wanted to know there was a reason for including it. Thanks!